packages/edit-site/src/components/dataviews/view-grid.js

```diff
@@ -32,41 +37,39 @@ export function ViewGrid( { data, fields, view, actions, getItemId } ) {
 			alignment="top"
 			className="dataviews-grid-view"
 		>
-			{ shownData.map( ( item, index ) => {
-				return (
-					<VStack key={ getItemId?.( item ) || index }>
-						<div className="dataviews-view-grid__media">
-							{ mediaField?.render( { item, view } ) || (
-								<Placeholder
-									withIllustration
```

Comment:
@jameskoster this used to render a Placeholder and now we just have the empty box-shadowed div. Is that the wanted design?